modules/gcp_helpers/authorization/main.tf

```diff
@@ -16,8 +16,9 @@ data "google_iam_role" "existing_org_role" {
 locals {
   has_specific_projects = length(var.projects) > 0
   has_excluded_projects = length(var.excluded_projects) > 0
-  filtered_projects     = local.has_excluded_projects ? [for project in data.google_projects.all.projects : project.project_id if !contains(var.excluded_projects, project.project_id)] : []
-  included_projects     = local.has_specific_projects ? var.projects : (local.has_excluded_projects ? local.filtered_projects : [])
+  filtered_projects     = local.has_excluded_projects ? [for project in data.google_projects.all.projects : project.project_id if !contains(var.excluded_projects, project.project_id)] : [for project in data.google_projects.all.projects : project.project_id]
```

Comment:
Why change the default case from [] to [for project in data.google_projects.all.projects : project.project_id]?